### PR TITLE
Adds initial spec for card component

### PR DESCRIPTION
**Pull request by procload**

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/06a52667968547f07b360da4d1d6f65891b7496f'>06a52667968547f07b360da4d1d6f65891b7496f</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/web-components/src/card/card.spec.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  - https://www.w3.org/WAI/ARIA/apg/patterns/slider/
+- [x] Are there any accessibility elements unique to this component?
+  - [x] No
+- [x] Does the component support 400% zoom?
+  - [x] Yes
+- [x] What keyboard behaviors does the component support?
+  - None yet, possibly some with selection
+
+## Preparation
+
```
Comment (mikeybkats) [inline packages/web-components/src/card/card.spec.md:55]: @procload I think the fluent card design guidelines should be here. did you review these as well? 

https://www.figma.com/file/8iZmitafCxS6LgKCCyDaFj/Card-templates?type=design&node-id=2254%3A57409&t=MAeL6uz4GuNizaRh-1

File: packages/web-components/src/card/card.spec.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
+# Slider
```
Comment (halerankin) [inline packages/web-components/src/card/card.spec.md:1]: Slider? :)

File: packages/web-components/src/card/card.spec.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+- preview: Used inside of a card and allows the user to better visualize what the card's contents are
+
+### CSS Variables
+
+None
+
+### Interactions
+
+- None yet
+- Will be adding the ability to select a card via an internal checkbox that renders when the card is set to selectable
```
Comment (halerankin) [inline packages/web-components/src/card/card.spec.md:41]: Will future plans include:
[Selectable](https://master--628d031b55e942004ac95df1.chromatic.com/?path=/docs/components-card-card--default#selectable)?
[Focus modes](https://master--628d031b55e942004ac95df1.chromatic.com/?path=/docs/components-card-card--default#focus-mode)?
[Draggable](https://www.figma.com/file/8iZmitafCxS6LgKCCyDaFj/Card-templates?type=design&node-id=2254-57414&t=hmPt88i6QvMhrrAp-4)?

**Review by halerankin — APPROVED** [review]

**Comment by layershifter** [comment]
Closing for housekeeping 🏡  Feel free to re-open 👍

**Pull request closed without merging**